fuse/utils/ndict.py

```diff
@@ -277,6 +277,17 @@ def get_multi(self, keys: Optional[List[str]]=None) -> NDict:
         
         return ans
 
+    def print_tree(self):
+        self.print_struct(self._stored)
+
+    def print_struct(self, d, level=0):
```

Comment:
Can you please add type annotation + comments?
If print_struct is for internal use, please add "_" prefix.
Use a longer name than "d",